requirements.txt

```diff
@@ -1,2 +1,6 @@
-Cython==0.28.2
-nose==1.3.0
+Cython
```

Comment:
I prefer specifying versions in requirements.txt. Are there any reasons to remove them?

Reply:
This is a mistake. All versions are now specified.